plugins/plugin-chart-echarts/src/Funnel/EchartsFunnel.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { EchartsProps } from '../types';
+import Echart from '../components/Echart';
+
+export default function EchartsFunnel({ height, width, echartOptions }: EchartsProps) {
```

Comment:
This component just passes through all its args,is there any reason not to use `<Echart>` instead of this?

Reply:
> This component just passes through all its args,is there any reason not to use `<Echart>` instead of this?

Thank you. I agree to use `<Echart>` to  instead of this very much personally for now.  BUT... I have a question.
Now `EchartsFunnel` is designed to meet the style of `BoxPlot`, `Pie` etc.  eg:
* https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/BoxPlot/EchartsBoxPlot.tsx
* https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/Gauge/EchartsGauge.tsx
* https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/Graph/EchartsGraph.tsx
* https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/Pie/EchartsPie.tsx
* https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

`EchartsFunnel` can be extended for future like `EchartsBoxPlot`.  BUT now,  Maybe there is no need to  extend `EchartsFunnel`

anyway. I want to code as the suggestion that is` use <Echart> instead of this`, but I want to meet the unified code style and designes too. SO I am hesitant for  a few days.
 I started to learn and code superset just for a few weeks. Can you provide more suggestion?

Reply:
Gotcha. Since there's an established pattern for the echarts, it probably does make sense to follow the pattern. I wasn't aware of this, haven't had a lot of exposure to the existing echarts solutions. Thanks for the informative response!

I'm still working my way through the rest of the changes, looking good so far!